internal/rego/oci/oci_test.go

```diff
@@ -184,6 +184,13 @@ func TestOCIDescriptorManifest(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
I didn't add test coverage for the happy path because the entry above already covers it.